Dockerfile.dyninst

```diff
@@ -0,0 +1,58 @@
+# Build stage with Spack pre-installed and ready to be used
+# docker build -f Dockerfile.dyninst -t ghcr.io/autamus/dyninst:05-25-master .
+FROM spack/ubuntu-bionic:latest as builder
+
+RUN git clone https://github.com/autamus/registry && \
+    cd registry && \
+    mv spack/d/dyninst /opt/spack/var/spack/repos/builtin/packages/dyninst
+
+# Install OS packages needed to build the software
+RUN apt-get -yqq update && apt-get -yqq upgrade \
+ && apt-get -yqq install python3-boto3 \
+ && rm -rf /var/lib/apt/lists/*
+
+# What we want to install and how we want to install it
+# is specified in a manifest file (spack.yaml)
+RUN mkdir /opt/spack-environment \
+&&  (echo "spack:" \
+&&   echo "  specs:" \
+&&   echo "  - dyninst@master" \
```

Comment:
@hainest this is what decides to build from the HEAD of dyninst on master.

Reply:
Ah, ok. I didn't notice this part!